framework/include/executioners/MultiSystemSolveObject.h

```diff
@@ -27,7 +27,7 @@ class MultiSystemSolveObject : public SolveObject
    * Returns a reference to the vector of solver systems that this object is
    * supposed to solve
    */
-  const std::vector<SolverSystem *> & systemsToSolve() { return _systems; }
+  const std::vector<SolverSystem *> & systemsToSolve() const { return _systems; }
```

Comment:
I don't think this should be regarded as const since it is returning pointers to non-const objects

Reply:
but calling the function does not modify the executioner?

Reply:
it returns a reference to data of the executuioner that a caller could modify. No different than a setter which we would not mark as const

Reply:
makes sense. I ll delete this one

Reply:
does it mean I need both then though?
Like if I have this:
```
class AA{
const Transient & exec;
checkSys()
{
for sys : exec.systemsToSolve()
...
}
```
I wont be able to call the non const one?

Reply:
You either make `_exec` non-const or you encapsulate your check in the `Executioner` class